src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1181,7 +1183,7 @@ && isCandidateContinuing(cvr.getCurrentRecipientOfVote())) {
           if (config.getMaxSkippedRanksAllowed() != Integer.MAX_VALUE
               && maxAllowedRanking - rank > config.getMaxSkippedRanksAllowed()) {
             recordSelectionForCastVoteRecord(
-                cvr, roundTally, null, StatusForRound.INACTIVE_BY_UNDERVOTE, "");
+                cvr, roundTally, null, StatusForRound.INACTIVE_BY_UNUSED_RANKINGS, "");
```

Comment:
I'm confused by this. The comments make this seem like this is from skipping too many ranks and should then be `INACTIVE_BY_SKIPPED_RANKING`? We look for skipped rankings elsewhere explicitly. Before it was treated as an undervote? Was that our strict 0 ranking undervote? But we can only get here if there is a rank...

Reply:
I have clarified the comment and ENUMs to determine if this is `EXHAUSTED_CHOICE_PARTIALLY_RANKED` or `EXHAUSTED_CHOICE_FULLY_RANKED`